docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
+=============
+
+.NET is a very flexible and naturally cross-platform [specification](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) for building developer products. It is used for all of the most popular app categories: desktop, mobile, cloud, gaming and IoT. 
+
+Major commercial and open source .NET products are listed below, in order of introduction.
+
+.NET Framework
+--------------
+
+The .NET Framework is Microsoft's primary develop platform for Windows server and client developers. It is a very powerful and mature framework, with a large class library that supports a wide variety of applications and solutions on Windows.
```

Comment:
Re: "Windows server and client developers" - I'd swap this to be "client and server" to match the usage.

"applications and solutions" feels strange since it's going from more specific to more broad in the same sentence.  Feels a bit more awkward when we later state that there are "multiple application models".  Perhaps drop "solutions"?